src/roles/hammer/README.md

```diff
@@ -0,0 +1,17 @@
+hammer role
```

Comment:
Should we make an issue to add README to each role?

Reply:
This was the result of a discussion @evgeni and I had. It's an effort to capture a design decission: we rely on Hammer itself to do the right thing in the role but in the playbook we combine them since there we know what it has to be. That way the role could be used standalone on other systems.

Reply:
Yeah, probably.

@ekohl and me discussed it'd be a good idea to point out how the roles are useable standalone and how they are integrated into foremanctl, so people can re-use them.

Reply:
It could even be in the foreman-operations-collection

Reply:
I think at some point operations and `src/roles` will just merge.

Reply:
Also what I'd like to see